xml/depl_maintenance.xml

```diff
@@ -391,17 +391,133 @@
       2017-02-02 - fs: What about SES-4 ?
      </remark>
      <simplelist>
-      <member>&cloud_repo;-Pool</member>
-      <member>&cloud_repo;-Update</member>
-      <member>&sle_repo;-Pool</member>
-      <member>&sle_repo;-Update</member>
-      <member>&sleha_repo;-Pool (for &hasetup;s only)</member>
-      <member>&sleha_repo;-Update (for &hasetup;s only)</member>
+      <member><literal>&cloud_repo;-Pool</literal></member>
+      <member><literal>&cloud_repo;-Update</literal></member>
+      <member><literal>&sle_repo;-Pool</literal></member>
+      <member><literal>&sle_repo;-Update</literal></member>
+      <member>
+       <literal>&sleha_repo;-Pool</literal> (for &hasetup;s only)
+      </member>
+      <member>
+       <literal>&sleha_repo;-Update</literal> (for &hasetup;s only)
+      </member>
      </simplelist>
      <para>
       Do not add these repositories to the &cloud; repository configuration,
       yet. This needs to be done during the upgrade procedure.
      </para>
+     <para>
+      If you have deployed &ceph; (&ses; 2.1) you also need to make the
+      &ses; 4 repositories available:
+     </para>
+     <simplelist>
+      <member><literal>&ses_repo;-Pool</literal></member>
+      <member><literal>&ses_repo;-Update</literal></member>
+     </simplelist>
+    </listitem>
+    <listitem>
+     <para>
+      Hybrid authentication with &o_ident; is not supported during upgrade. If
+      you have configured this authentication method (see <citetitle>Using
+      Hybrid Authentication</citetitle> in the &productname; 6 Deplyoment
+      Guide), you need to revert this setting prior to starting the upgrade. To
+      do so, open the &o_ident; &barcl; <guimenu>Attribute</guimenu>
+      configuration in <guimenu>Raw</guimenu> mode. Set the identity and
+      assignment drivers as follows:
+     </para>
+     <screen>  "identity": {
+    "driver": "sql"
+  },
+  "assignment": {
+    "driver": "sql"
+  },</screen>
+     <para>
+      After the upgrade has finished, re-enable hybrid authentication as
+      described in <xref linkend="sec.depl.ostack.keystone.ldap.hybrid"/>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      &productname; 7 comes with a new version of &o_objstore;. Replica
+      handling has changed with this version&mdash;the number of replicas
+      cannot be higher than the number of available disks. Check the
+      <guimenu>Replica</guimenu> setting in the &o_objstore; &barcl; prior to
+      starting the upgrade. If it is higher than the number of available disks,
+      adjust it to match the number of disks and redeploy the &barcl;.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Upgrading is not supported if &o_blockstore; has been deployed with the
+      <literal>raw devices</literal> back-end.
```

Comment:
This is not exactly true. For **non-disruptive upgrade** cinder backend cannot be raw devices or local file. Normal upgrade can use them